### PR TITLE
fix: Replace vb on vbc/cl/csc/svcutil code fences

### PR DESCRIPTION
**Pull request by nschonni**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/standard/assembly/sign-strong-name.md [hunk 1]
```diff
@@ -19,63 +19,63 @@ dev_langs:
...
-At the [Developer Command Prompt for Visual Studio](../../framework/tools/developer-command-prompt-for-vs.md), enter the following command:  
+> In order to [delay sign an assembly](delay-sign.md), choose a public key file.
+
+### Create and sign an assembly with a strong name by using the Assembly Linker
 
-**al** **/out:**\<*assemblyName*> *\<moduleName>* **/keyfile:**\<*keyfileName*>  
+At the [Developer Command Prompt for Visual Studio](../../framework/tools/developer-command-prompt-for-vs.md), enter the following command:
 
-Where:  
+**al** **/out:**\<*assemblyName*> *\<moduleName>* **/keyfile:**\<*keyfileName*>
```
Comment (Thraka) [inline docs/standard/assembly/sign-strong-name.md]:
```suggestion
`al /out:\<assemblyName> \<moduleName> /keyfile:\<keyfileName>`
```

**Review by Thraka — CHANGES_REQUESTED** [review]
Looks great! One minor change. These format of code example where the parameters are broken up into italic/bold to indicate something is no longer used. It was ported from MSDN. I've fixed that one.

File: docs/framework/app-domains/build-multifile-assembly.md [hunk 2]
```diff
@@ -81,15 +81,15 @@ This article explains how to create a multifile assembly and provides code that
 
 2. Use the following command to compile this code:
 
-   ```cpp
+   ```console
```
Comment (mairaw) [inline docs/framework/app-domains/build-multifile-assembly.md]: If we change these to console, all three will show up for any selected language and confuse costumers.

**Review by mairaw — CHANGES_REQUESTED** [review]
I'm still not sure if changing to console on the multi-language topics is the right thing to do.

File: docs/framework/app-domains/build-multifile-assembly.md (hunk 2 above)
Comment (mairaw) [reply docs/framework/app-domains/build-multifile-assembly.md]: yeah, to change that to console, we'd have to use tabs. not sure if it's worth it in these topics.
There's no support for double identifiers.

**Pull request closed without merging**